webapp/client/src/components/FormFieldCheckBox.test.js

```diff
@@ -0,0 +1,67 @@
+import React from "react";
+import { render, screen } from "@testing-library/react";
+import userEvent from "@testing-library/user-event";
+import FormFieldCheckBox from "./FormFieldCheckBox";
+
+describe("When FormFieldCheckBox with default values", () => {
+  let props;
+
+  beforeEach(() => {
+    props = {
+      fieldName: "fooName",
+      fieldId: "fooId",
+      labelText: "foo",
+      errors: [],
```

Comment:
Do we need to test that errors are shown?
And I think we never test for a simple click() event?
How about "Shows label"?

Reply:
> How about "Shows label"?

I do not know if we need to test for that because we select the checkBox through the labelText?